superset-frontend/src/views/CRUD/hooks.ts

```diff
@@ -45,11 +46,12 @@ export function useListViewResource<D extends object = any>(
   infoEnable = true,
   defaultCollectionValue: D[] = [],
   baseFilters?: FilterValue[], // must be memoized
+  defaultLoading = true,
```

Comment:
Nit: can we call it `initialLoadingState`?  This function also has too many positional arguments. In general, I'd recommend refactoring any function with >=3 arguments to use `options` object.

Reply:
Yep, I'll do the refactoring in a follow up ticket.